karma.conf.js

```diff
@@ -1,21 +1,13 @@
 var Webpack        = require('webpack')
 var webpack_config = require('./webpack.config')
-var isMac          = require('os').platform() === 'darwin'
 
 module.exports = function(config) {
-  var browsers = [ 'Chrome', 'Firefox' ]
-
-  if (isMac) {
-    browsers.push('Safari')
-  }
-
   config.set({
-
-    browsers: browsers,
+    browsers: [ 'Chrome' ],
 
     frameworks: [ 'mocha', 'sinon-chai' ],
 
-    logLevel: config.LOG_ERROR,
+    autoWatchBatchDelay: 400,
```

Comment:
@greypants: This _might_ address the issue you were bumping into with the build compiling twice. It's working for me, at any rate.